src/sniffer.c

```diff
@@ -341,6 +345,27 @@ typedef struct FinCaputre {
 } FinCaputre;
 
 
+typedef struct HsHashes {
+#ifndef NO_OLD_TLS
+#ifndef NO_SHA
+    Sha hashSha;
+#endif
+#ifndef NO_MD5
+    Md5 hashMd5;
+#endif
+#endif
+#ifndef NO_SHA256
+    Sha256 hashSha256;
+#endif
+#ifdef WOLFSSL_SHA384
+    Sha384 hashSha384;
+#endif
+#ifdef WOLFSSL_SHA512
+    Sha512 hashSha512;
```

Comment:
No. I copied the Sha512 part of the hashing from src/internal.c. We need it there because of the signatures in the certificate verify and server key exchange. The cipher suites themselves don't use SHA-512 and there isn't a possibility of it getting checked here.